pkg/sos/html/wifi.html

```diff
@@ -0,0 +1,154 @@
+<head>
+	<script>
```

Comment:
We should start thinking about separating out the javascript into different files. I'm not sure we want to keep embedding it in the html. This is fine for now though. Not a blocker.

Reply:
I agree. That's the next step after getting upspin up to speed with the other pages